proto/grp.proto

```diff
@@ -99,3 +99,10 @@ message UpdateGroupRequest {
 }
 
 message UpdateGroupResponse {}
+
+enum GroupMembershipStatus {
```

Comment:
nit: add an unknown state as 0

enum GroupMembershipStatus {
  UNKNOWN_MEMBERSHIP_STATUS = 0;

  // The user is a member of the group.
  MEMBER = 1;

  // The user has requested to join the group but is not yet a member.
  REQUESTED = 2;
}